lib/g_places.js

```diff
@@ -0,0 +1,83 @@
+https = require('https');
+
+/**
+ * function to search google places
+ * @param  {[object] name, lat, long from input form}
+ * @return {[object] gp - literal with google place values for newRest }
+ */
+module.exports = function(params, callback) {
+
+  var apikey = 'AIzaSyCY1BQZJ8B4_QjNr-UDFGvtInFs9SF7JM0';
```

Comment:
i mostly just meant the code. probably not a huge deal if someone sees our api key for this, but still a good practice to keep it private i think.